data/tokens.json

```diff
@@ -3021,6 +3021,253 @@
       "type": "spacing"
     }
   },
+  "inputs": {
+    "gtmds_input_size-height-sm": {
+      "value": "{gtmds_size-lg}*2",
+      "type": "sizing"
+    },
+    "gtmds_input_size-height-lg": {
+      "value": "{gtmds_size-xl}+{gtmds_size-lg}",
+      "type": "sizing"
+    },
+    "gtmds_input_size-width": {
+      "value": "auto",
+      "type": "sizing"
+    },
+    "gtmds_input_margin-right": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "gtmds_input-content_margin-right": {
+      "value": "{gtmds_space-xxs}/2",
+      "type": "spacing"
+    },
+    "gtmds_input_margin-bottom": {
+      "value": "{gtmds_space-xxs}/2",
+      "type": "spacing"
+    },
+    "gtmds_input_padding-left": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "gtmds_input_padding-right": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "gtmds_input_padding-vertical-sm": {
+      "value": "{gtmds_space-xxs}/2",
+      "type": "spacing"
+    },
+    "gtmds_input_padding-vertical-lg": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "Chip": {
+      "gtmds_input_margin-row-bottom": {
+        "value": "{gtmds_space-xxs}/2",
+        "type": "spacing"
+      },
+      "gtmds_input_margin-row-right": {
+        "value": "{gtmds_space-xxs}/2",
+        "type": "spacing"
+      },
+      "gtmds_input_padding-left-sm": {
```

Comment:
what do we use this token for?

Reply:
It is in the "Chip" group, where it defines the left padding when chips are selected. It is reduced because it looks odd otherwise.

Reply:
I cannot find it in the file can you please point me with a comment in Figma?